cmd/kubeadm/app/util/version_test.go

```diff
@@ -165,3 +165,36 @@ func TestVersionToTag(t *testing.T) {
 		}
 	}
 }
+
+func TestSplitVersion(t *testing.T) {
+	type T struct {
+		input  string
+		bucket string
+		label  string
+		valid  bool
+	}
+	cases := []T{
+		{"v1.7.0", "foo/release", "v1.7.0", true},
+		{"release/v1.7.0", "foo/release", "v1.7.0", true},
+		{"release/latest-1.7", "foo/release", "latest-1.7", true},
+		{"ci/latest-1.7", "foo/ci-cross", "latest-1.7", true},
```

Comment:
yes, they are the same, but we should show that in unit tests that it's expected as well ;)